### ISSUE TITLE
10th April Deadline

### ISSUE THREAD

**Issue by snuff4** [rishabhj126/Institute_login#1]
# 10th April
- [ ] Simple Layout 
- [ ] A unified login system for now, consisting of register/login
- [ ] A class to store these values and check against them
- [ ] If verified, taken to the dashboard
- [ ] Dashboard will consist of 1st year (for now)
- [ ] Clicking 1st year takes to the schedule
- [ ] And clicking the day takes to the lectures for the day

We will not implement the teacher's part as of now. That will come in the new issue.

**Comment by blackpanther99** [comment]
I have done the first two parts.. ihave linked student and teacher layouts with their login credentials... I will need help in designing the timetable and lectures of the day

**Comment by snuff4** [comment]
@blackpanther99 Yes. I will read about how to work with the table views in Android, and revert back to you by today.
